iot_control/models/iot_partner.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2022 thingsintouch.com
+# Copyright 2022 ForgeFlow S.L. (https://www.forgeflow.com)
+# Copyright 2022 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class PartnerIoT(models.Model):
+    _name = "iot.partner"
+    _description = "Person with Control on IoT Devices"
+
+    _sql_constraints = [
+        ("rfid_code_uniq", "UNIQUE(rfid_code)", "The rfid code should be unique.",)
+    ]
+
+    partner_id = fields.Many2one(
```

Comment:
Relate using res_model and res_id, similar to `mail.message`

Also, a mixin similar to `mail.thread` might be interesting to relate to this kind of records.

Reply:
@etobella , do you want this to relate an iot.partner to any model right? so iot.partner can be a res.partner a hr_employee or whatever. Is that correct?